### DISCUSSION TITLE
[Script request] Dolibarr ERP & CRM

### DISCUSSION THREAD

**Discussion by 391a** [community-scripts/ProxmoxVE#973] [Request script] [answered]
### Application Name

Dolibarr

### Website

https://www.dolibarr.org/

### Description

Dolibarr ERP & CRM is a modern software package that helps manage your organization's activities (contacts, quotes, invoices, orders, stocks, agenda, human resources, ecm, manufacturing…).
https://github.com/Dolibarr/dolibarr

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by bbha01** [comment]
I would love to see this too!

**Comment by tgrignon** [comment]
I want !

**Comment by tremor021** [comment, marked as answer]
#2502

**Comment by mccorriente** [comment]
<img width="520" height="326" alt="image" src="https://github.com/user-attachments/assets/b836d10e-0684-4994-9633-32c3c3602ccd" />

The automatic script gives me this error.... helpppp
